package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "perenv.macro",
-  "version": "0.0.2",
+  "version": "0.0.0-semver",
```

Comment:
That's strange.. what is `-semver`? semantic version?

Reply:
Yeah, the release version is set at the time of publish. Doesn't make sense to make it part of the pull request.